libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -197,15 +189,20 @@ - (void)initializeAppViewState
     self.window.rootViewController = self.viewController;
     [self.window makeKeyAndVisible];
 }
-                                                            
 
-@end
+- (void)logWithLevel:(DDLogLevel)level format:(NSString *)format, ... {
```

Comment:
Not a bad idea. I could do that. However, this will change the behavior here. I was logging them all with different component names. This will make them all use the same component name - `default`. Is that what we want for tests? I guess it's not a big deal - they're just tests.